cli/dcoscli/marathon/main.py

```diff
@@ -105,17 +105,23 @@
 from dcos import cmds, emitting, jsonitem, marathon, options, util
 from dcos.errors import DCOSException
 from dcoscli import tables
+from dcoscli.main import handle_docopt_error
 
 logger = util.get_logger(__name__)
 emitter = emitting.FlatEmitter()
 
 
+@handle_docopt_error
 def main():
     try:
         return _main()
     except DCOSException as e:
         emitter.publish(e)
         return 1
+    except docopt.DocoptExit as e:
+        emitter.publish("Command Not Recognised. Please see Usage")
```

Comment:
You can remove "Please see Usage" from the error message since this is explicit (it prints Usage below).